addons/website/static/tests/tours/powerbox_snippet.js

```diff
@@ -0,0 +1,53 @@
+import wTourUtils from "@website/js/tours/tour_utils";
+
+wTourUtils.registerWebsitePreviewTour("website_powerbox_snippet", {
+    edition: true,
+    test: true,
+}, () => [
+    wTourUtils.dragNDrop({
+        id: "s_text_block",
+        name: "Text",
+    }),
+    {
+        content: "Check if s_text_block snippet is inserted",
+        trigger: "iframe .s_text_block",
+    }, {
+        content: "Select the last paragraph",
+        trigger: "iframe .s_text_block p:last-child",
+        run: "click",
+    }, {
+        content: "Show the powerbox",
+        trigger: "iframe .s_text_block p:last-child",
+        run: (actions) => {
+            actions.text(`/`, this.$anchor[0]);
+            const wrapwrap = this.$anchor[0].closest("#wrapwrap");
+            wrapwrap.dispatchEvent(
+                new InputEvent("input", {
+                    inputType: "insertText",
+                    data: "/",
+                })
+            );
+        },
+    }, {
+        content: "Change the content to '/banner'",
+        trigger: "iframe .s_text_block p:last-child",
+        run: () => {
+            const wrapwrap = this.$anchor[0].closest("#wrapwrap");
+            this.$anchor[0].textContent = "/banner";
+            wrapwrap.ownerDocument.dispatchEvent(
+                new KeyboardEvent('keyup', {
+                    key: 'DummyKey',
+                    code: 'KeyDummy',
+                    cancelable: true,
+                })
+            );
+        },
+    }, {
```

Comment:
May be you should add a step to just verify whether alert command is present or not in power box after search.
With no search, it will be also present in the powerbox & tour will pass.

Reply:
@divy-odoo  i have pushed the requested changes.